benchmark/Microsoft.IdentityModel.Benchmarks/CreateTokenTests.cs

```diff
@@ -19,16 +20,32 @@ public class CreateTokenTests
         [GlobalSetup]
         public void Setup()
         {
+            DateTime now = DateTime.UtcNow;
             _jsonWebTokenHandler = new JsonWebTokenHandler();
             _tokenDescriptor = new SecurityTokenDescriptor
             {
-                Subject = new ClaimsIdentity(Default.PayloadClaims),
+                Audience = Default.Audience,
+                Expires = now + TimeSpan.FromDays(1),
+                Claims = new Dictionary<string, object>
+                {
+                    { JwtRegisteredClaimNames.Azp, Default.Azp },
+                    { JwtRegisteredClaimNames.Email, "Bob@contoso.com" },
+                    { JwtRegisteredClaimNames.Jti, Default.Jti },
+                },
+                Issuer = Default.Issuer,
+                IssuedAt = now,
+                NotBefore = now,
                 SigningCredentials = KeyingMaterial.JsonWebKeyRsa256SigningCredentials,
             };
         }
 
         [Benchmark]
-        public string JsonWebTokenHandler_CreateToken() => _jsonWebTokenHandler.CreateToken(_tokenDescriptor);
+        public string JsonWebTokenHandler_CreateToken()
+        {
+            for (int i = 0; i< 1000; i++)
```

Comment:
this confuses me. Benchmark dotnet already does many iterations. Is this just to be a scalar on top of that? Why not just specify the iterations?

Reply:
@keegan-caruso good point, i was just checking will remove this code.